news/3995.breaking.2

```diff
@@ -0,0 +1,2 @@
+Moved plone.protect.confirm template to plone.app.layout.views
```

Comment:
I would be more descriptive.
What a bout something like:
```suggestion
Provide a Classic UI speficic template for the `plone.protect` `@@confirm-action`  view.
```

But:

1. I am not good in this things, @stevepiercy is my hero here :robot: 
2. I do not consider this a blocker